src/env.rs

```diff
@@ -0,0 +1,36 @@
+//! Fns to read variables from the environment more conveniently and help other functions figure
+//! out what environment they're running in.
+
+use std::env;
+
+use tracing::debug;
+
+const SECRET_LOG_BLACKLIST: [&str; 3] = ["DATABASE_URL", "OPSGENIE_API_KEY", "ETHERSCAN_API_KEY"];
+
+/// Get an environment variable, encoding found or missing as Option, and panic otherwise.
+pub fn get_env_var(key: &str) -> Option<String> {
```

Comment:
Not the prettiest code I've written, but it works 😄 .

Reply:
I've definetely written uglier code in my life. (That didn't even work properly😅)